zproject/prod_settings_template.py

```diff
@@ -723,6 +723,18 @@
 ## set big_blue_button_secret in zulip-secrets.conf.
 # BIG_BLUE_BUTTON_URL = "https://bbb.example.com/bigbluebutton/"
 
+################
+## LLM Summarization
+##
+## The model name that will used by LiteLLM library to configure
+## parameters to be sent to API.
+## The Llama-3-8B-instruct model is free to use and only requires submitting
+## a small form on the HuggingFace page for the model to gain access.
+# TOPIC_SUMMARIZATION_MODEL = "huggingface/meta-llama/Meta-Llama-3-8B-Instruct"
+
+## Other configuration parameters, passed through to litellm's `completion` call
+## See https://docs.litellm.ai/docs/completion/input
+# TOPIC_SUMMARIZATION_PARAMETERS = {}
```

Comment:
Reading this, I don't think we need a `TOPIC_SUMMARIZATION_MODEL` setting at all; `model` would be just one of `TOPIC_SUMMARIZATION_PARAMETERS`. 

The other thing on my mind is that the `INPUT_COST_PER_GIGATOKEN` and `OUTPUT_COST_PER_GIGATOKEN` fields belong as settings stored with the choice of model and provider. Do we want to make those optional keys in `TOPIC_SUMMARIZATION_PARAMETERS` that get popped off in `computed_settings.py` or `message_summary.py` so as to not send into `litellm` itself?

Reply:
I think I still want a single configuration dictionary for a given model. Like, here, we do need 3 settings -- the two cost ones and the model name.